likelihood/cl_like/cl_like.py

```diff
@@ -70,7 +70,7 @@ def get_lmax_from_kmax(cosmo, kmax, z, nz):
                 raise LoggedError(self.log, "Unknown tracer %s" % b['name'])
             t = s.tracers[b['name']]
             self.bin_properties[b['name']] = {'z_fid': t.z,
-                                              'nz_fid': t.nz}
+                    'nz_fid': t.nz,'zmean_fid': sum(t.z*t.nz)/sum(t.nz)}
```

Comment:
A more succinct version of this would be `'zmean_fid': np.average(t.z, weights=t.nz)`
Also, nerdy request: can you break this into two lines and make sure they're aligned with `'z_fid'` in the previous line?